### ISSUE TITLE
[Bug] Default `vpc-cni` add-on version not found when creating clusters

### ISSUE THREAD

**Issue by etungsten** [weaveworks/eksctl#5074]
### What were you trying to accomplish?

I'm trying to create a new IPv6 EKS 1.22 cluster.

### What happened?

Cluster creation fails after trying to set up vpc-cni add-on.
```
2022-04-05 12:10:20 [ℹ]  waiting for CloudFormation stack "eksctl-etung-1-22-ipv6-cluster"
2022-04-05 12:11:20 [ℹ]  waiting for CloudFormation stack "eksctl-etung-1-22-ipv6-cluster"
2022-04-05 12:12:21 [ℹ]  waiting for CloudFormation stack "eksctl-etung-1-22-ipv6-cluster"
2022-04-05 12:14:24 [ℹ]  daemonset "kube-system/aws-node" restarted
2022-04-05 12:14:25 [!]  1 error(s) occurred and cluster hasn't been created properly, you may wish to check CloudFormation console
2022-04-05 12:14:25 [ℹ]  to cleanup resources, run 'eksctl delete cluster --region=us-west-2 --name=etung-1-22-ipv6'
2022-04-05 12:14:25 [✖]  failed to fetch version  for addon vpc-cni: no version(s) found matching "1.10.0" for "vpc-cni"
```

### How to reproduce it?

Create a cluster with the following cluster configuration:
```
apiVersion: eksctl.io/v1alpha5
kind: ClusterConfig
metadata:
  name: "etung-1-22-ipv6"
  version: "1.22"
  region: "us-west-2"
kubernetesNetworkConfig:
  ipFamily: IPv6
vpc:
  clusterEndpoints:
    publicAccess: true
    privateAccess: true
iam:
  withOIDC: true
addons:
  - name: vpc-cni
  - name: coredns
  - name: kube-proxy
managedNodeGroups:
  - name: br-mng
    amiFamily: Bottlerocket
    availabilityZones: []
    minSize: 0
    desiredCapacity: 0
    maxSize: 1
```
Then ran `eksctl create cluster -f "${cluster_config_file}" --set-kubeconfig-context=false`

**Logs**

**Anything else we need to know?**

Using eksctl version 0.92.0-rc.0, but this happens with other versions and cluster versions as well.
eksctl documentation says the versions for the add-ons are optional and is set to a default version for a given cluster version.
This could also be because I didn't create a worker node as part of the cluster creation and only created an empty managed nodegroup. But the error message indicates otherwise.

**Versions**

```
$ eksctl info
```
```
$ eksctl info
eksctl version: 0.92.0-rc.0
kubectl version: v1.21.0
OS: linux
```

**Comment by aclevername** [comment]
Thanks for opening the issue @etungsten , I've managed to reproduce this and am taking a look at a fix

**Comment by aclevername** [comment]
I have opened a PR to fix the issue #5078, @etungsten in the meantime a simple workaround would be to set the version to `version: latest` for the vpc-cni, or any specific version you like :smile:
